homepage.php

```diff
@@ -49,57 +49,57 @@
 		<div class="content_4">
 			<Space></Space>
 			<div class="images1">
-				<a href = "Query8.php"><img src = "images/Indie2.jpg" class="image1"></a>
+				<a href = "Query8.php"><img src = "images/Indie.jpg" class="image1"></a>
```

Comment:
14/09/2021: I trialing for some img for it my banner for the home page. To find the img that could match with the theme of my music database and also it should look nice so the first impression of the user for the database will be good.